### PR TITLE
Make Locale match defaultLocale

### LINKED ISSUE #174424: [Docs] Unclear when defaultLocale PackageName/Publisher started affecting ARP entry correlation
For the fields `PackageName` and `Publisher` in the Locale schema, since schema 1.0.0, there has consistently been a note like the following (modulo irrelevant phrasing tweaks regarding Windows 11 and/or the Settings app and/or prefixing "entry" with "ARP") regarding a semantic change in winget 1.0:

> [!NOTE]
> With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP) and Windows Apps & Features respectively. The best practice is to ensure this matches the ARP entry for the package name when it has been installed.

In the docs for all subsequent schema versions that still exist, those notes continue to reference Windows Package Manager v1.0; presumably that's when that change to Locale semantics was made.

However, the same fields in the **defaultLocale** schema have near-identical notes, except the Windows Package Manager version isn't frozen in their case:

| Schema Version | Windows Package Manager Version |
| --- | --- |
| 1.0.0 | 1.0 |
| 1.1.0 | 1.2 |
| 1.2.0 | 1.3 |
| 1.4.0 | 1.4 |
| 1.5.0 | 1.5 |
| 1.6.0 | 1.6 |
| 1.7.0 | 1.7 |

Ought these notes all refer to 1.0 instead, or was there actual intentionality behind any of the incrementations?

### PR DESCRIPTION
**Pull request by Trenly**
* Resolves #174424
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/174440)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: doc/manifest/schema/1.7.0/locale.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ ManifestVersion: 1.7.0        # The manifest syntax version
 
   This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
 
-  NOTE:	With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  NOTE:	With the 1.7 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.7.0/locale.md:136]:
```suggestion
> [!Note]
> With the 1.7 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.7.0/locale.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ ManifestVersion: 1.7.0        # The manifest syntax version
 
   This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
 
-  NOTE:	With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  NOTE:	With the 1.7 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.7.0/locale.md:94]:
```suggestion
> [!Note]
> With the 1.7 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.6.0/locale.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ ManifestVersion: 1.6.0        # The manifest syntax version
 
   This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
 
-  NOTE:	With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  NOTE:	With the 1.6 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.6.0/locale.md:136]:
```suggestion
> [!Note]
> With the 1.6 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.6.0/locale.md [hunk 4]
```diff
@@ -91,7 +91,7 @@ ManifestVersion: 1.6.0        # The manifest syntax version
 
   This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
 
-  NOTE:	With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  NOTE:	With the 1.6 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.6.0/locale.md:94]:
```suggestion
> [!Note]
> With the 1.6 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.5.0/locale.md [hunk 5]
```diff
@@ -136,7 +136,7 @@ ManifestVersion: 1.5.0        # The manifest syntax version
   This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
 
   > [!NOTE]
-	With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+	With the 1.5 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.5.0/locale.md:139]:
```suggestion
> With the 1.5 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.5.0/locale.md [hunk 6]
```diff
@@ -93,7 +93,7 @@ ManifestVersion: 1.5.0        # The manifest syntax version
   This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
 
   > [!NOTE]
-	With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+	With the 1.5 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.5.0/locale.md:96]:
```suggestion
> With the 1.5 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.4.0/locale.md [hunk 7]
```diff
@@ -133,7 +133,7 @@ ManifestVersion: 1.4.0        # The manifest syntax version
 
   This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
 
-  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  >Note: With the 1.4 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.4.0/locale.md:136]:
```suggestion
> [!Note]
> With the 1.4 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.4.0/locale.md [hunk 8]
```diff
@@ -91,7 +91,7 @@ ManifestVersion: 1.4.0        # The manifest syntax version
 
   This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
 
-  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  >Note: With the 1.4 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.4.0/locale.md:94]:
```suggestion
> [!Note]
> With the 1.4 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.2.0/locale.md [hunk 9]
```diff
@@ -133,7 +133,7 @@ ManifestVersion: 1.2.0        # The manifest syntax version
 
   This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
 
-  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  >Note: With the 1.2 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.2.0/locale.md:136]:
```suggestion
> [!Note]
> With the 1.2 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.2.0/locale.md [hunk 10]
```diff
@@ -91,7 +91,7 @@ ManifestVersion: 1.2.0        # The manifest syntax version
 
   This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
 
-  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  >Note: With the 1.2 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.2.0/locale.md:94]:
```suggestion
> [!Note]
> With the 1.2 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.1.0/locale.md [hunk 11]
```diff
@@ -128,7 +128,7 @@ ManifestVersion: 1.1.0        # The manifest syntax version
 
   This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
 
-  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  >Note: With the 1.1 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.1.0/locale.md:131]:
```suggestion
> [!Note]
> With the 1.1 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

File: doc/manifest/schema/1.1.0/locale.md [hunk 12]
```diff
@@ -86,7 +86,7 @@ ManifestVersion: 1.1.0        # The manifest syntax version
 
   This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
 
-  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+  >Note: With the 1.1 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```
Comment (stephengillie) [inline doc/manifest/schema/1.1.0/locale.md:89]:
```suggestion
> [!Note]
> With the 1.1 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
@stephengillie - I'm going to reject your suggestions in this PR and raise a separate PR to adjust that formatting, as there are many other places these notes occurr in the docs, including files not touched by this PR.

Please merge.

**Review by ItzLevvie — APPROVED** [review]

File: doc/manifest/schema/1.7.0/locale.md (hunk 1 above)
Comment (cvrebert) [reply doc/manifest/schema/1.7.0/locale.md:136]: The special alert syntax for notes etc. doesn't work within `<details>` elements (or nested within any element); see #174228

**Comment by Trenly** [comment]
[Policy] Reset Feedback

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
It seems that https://github.com/microsoft/winget-pkgs/labels/Project-File PRs don't pass pipelines, so never get the https://github.com/microsoft/winget-pkgs/labels/Validation-Completed label, creating a situation where the Policy Service doesn't have the right labels to make an approval.

**Pull request merged**